mdagent/tools/base_tools/analysis_tools/rmsd_tools.py

```diff
@@ -257,13 +277,17 @@ def _run(
         plot: bool = True,
     ):
         try:
-            rmsd = RMSDFunctions(pdb_file, trajectory, ref_file, ref_trajectory)
+            rmsd = RMSDFunctions(
+                self.path_registry, pdb_file, trajectory, ref_file, ref_trajectory
+            )
             message = rmsd.calculate_rmsd(rmsd_type, selection, plot)
         except ValueError as e:
             return (
                 f"ValueError: {e}. \nMake sure to provide valid PBD "
                 "file and binding site using MDAnalysis selection syntax."
             )
+        except FileNotFoundError as e:
+            return str(e)
```

Comment:
yeah, i like to make this messages to the agent more descriptive.

Reply:
I add the description above where I define the filenotfound error message